scripts/build-cdn-indexes.ts

```diff
@@ -53,12 +52,12 @@ getSystems().forEach((sysinfo) => {
           <ul>
             <li>Main JS Bundle:<br> <code class="ds-u-fill--gray-lightest ds-u-padding--1 ds-u-margin--1 ds-u-display--inline-block"><a href="https://design.cms.gov/cdn/${system}/${version}/js/bundle.js">https://design.cms.gov/cdn/${system}/${version}/js/bundle.js</a></code></li>
```

Comment:
The react version also changed, and it needs to be updated in the HEAD of the document too. All of them are now under `*-components/bundle/*.js`